Get-LockboxAudits.ps1

```diff
@@ -118,11 +128,12 @@ function Get-Audits-For-Users-In-Workspace{
                 catch {
                     if ($retryCount -gt 3){
                         Write-Host "Could not send Information after 4 retrys."
-                        Write-Host $usernameBody

```

Comment:
this line is still useful for debugging purpose.  maybe put it to Write-Debug or Write-Verbose?

Reply:
Will add back.